docs/output.md

```diff
@@ -697,7 +697,7 @@ This table shows the contents of the `snpStatistics.tsv` file produced by MultiV
 
 You can get different variants of the Call statistics bar plot, depending on how you configured  the MultiVCFAnalyzer options.
 
-If you ran with `--min_allele_freq_hom` and `--min_allele_freq_het` set to two different values (left panel A in the figure below), this allows you to assess the number of multi-allelic positions that were called in your genome. Typically MultiVCFAnalyzer is used for analysing smallish on haploid genomes (such as bacteria), therefore a position with multiple possible 'alleles' suggests some form of cross-mapping from other taxa. If this is the case, you will need to be careful with downstream analysis of the consensus sequence (e.g. for phylogenetic tree analysis) as you may accidently pick up SNPs from other taxa - particularly when dealing with low coverage data. Therefore, if you have a high level of 'het' values, you should carefully check your alignments manually to see how clean your genomes are, or whether you can do some form of strain separation (e.g. by majority/minor calling).
+If you ran with `--min_allele_freq_hom` and `--min_allele_freq_het` set to two different values (left panel A in the figure below), this allows you to assess the number of multi-allelic positions that were called in your genome. Typically MultiVCFAnalyzer is used for analysing smallish on haploid genomes (such as bacteria), therefore a position with multiple possible 'alleles' suggests some form of cross-mapping from other taxa. If this is the case, you will need to be careful with downstream analysis of the consensus sequence (e.g. for phylogenetic tree analysis) as you may accidentally pick up SNPs from other taxa - particularly when dealing with low coverage data. Therefore, if you have a high level of 'het' values, you should carefully check your alignments manually to see how clean your genomes are, or whether you can do some form of strain separation (e.g. by majority/minor calling).
```

Comment:
I don't think I understand what "smallish on haploid genomes"... what does that mean? multiple possible 'alleles' can also suggest multiple strains being present. The shape of the plot (do we have any plot being produced???) can give you an indication of that